api/src/pcapi/routes/backoffice_v3/pro_users/blueprint.py

```diff
@@ -152,7 +152,9 @@ def delete(user_id: int) -> utils.BackofficeResponse:
         return redirect(url_for("backoffice_v3_web.pro_user.get", user_id=user_id), code=303)
 
     # clear from mailing list
-    mails_api.delete_contact(user.email)
+    if not offerers_models.Venue.query.filter(offerers_models.Venue.bookingEmail == user.email).one_or_none():
```

Comment:
attention si il y en a plusieurs ça risque d'exploser (plusieurs venues peuvent utiliser la même adresse email)
une solution plus safe et plus rapide (et qui retournera que 1 ou 0)
```suggestion
    if not offerers_models.Venue.query.filter(offerers_models.Venue.bookingEmail == user.email).limit(1).count():
```

On pourrait optimiser encore plus mais je ne sui pas certain que ce soit indispensable